python/tox.ini

```
@@ -35,7 +35,7 @@ omit =
   src/aac/ui/*
 
 [report]
-fail_under = 91.75
+fail_under = 92.05
```

I think we've reached a really good coverage at this point. Going for higher coverage will likely lead us into some bad and less useful testing practices. Personally, I think 90 is a good round number, but 92% is also good.
```suggestion
fail_under = 92.00
```